kahuna/public/js/search/index.js

```diff
@@ -149,7 +149,19 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
 
                     const sub = dragData$.subscribe(({ images, dt }) => {
                         if (images.size > 0) {
+                            //creates an element to use as the drag icon
+                            const dragImage = document.createElement('div');
+                                  dragImage.classList.add("dragIcon");
+
+                            const imageCount = document.createElement('span');
+                                  imageCount.classList.add("dragCount");
+
+                                  imageCount.innerHTML = images.count();
+                            dragImage.appendChild(imageCount);
+                            document.body.appendChild(dragImage);
```

Comment:
@tsop14 [I've made it attach to the parent element so that it's automatically garbage collected](https://github.com/guardian/grid/commit/c7e78edd9c49663573b457a07d4a9e582a320795).
@theefer it seems you do need to have it attached if it's a DOM element.